lib/logstash/outputs/elasticsearch_java.rb

```diff
@@ -162,37 +165,24 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   #     `["127.0.0.1:9200","127.0.0.2:9200"]`
   # It is important to exclude http://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html[dedicated master nodes] from the `host` list
   # to prevent LS from sending bulk requests to the master nodes.  So this parameter should only reference either data or client nodes.
-
-  config :host, :validate => :array
+  config :hosts, :validate => :array, :default => ["127.0.0.1"]
```

Comment:
I think historically we've allowed `:host` (singular) to be an `:array`. Thoughts on keeping it the same singular? Thoughts on that style?